packages/webapp/webapp_server.js

```diff
@@ -435,7 +433,7 @@ WebAppInternals.generateBoilerplateInstance = function (arch,
     ...__meteor_runtime_config__,
     ...(additionalOptions.runtimeConfigOverrides || {})
   };
-  runtimeConfig.updateHooks.forEach((cb) => {
+  runtimeConfig.updateHooks.each((cb) => {
```

Comment:
You can use here `forEach`, it has been added in `callback-hook@1.4.0` which is available on the release branch.